deployment/scripts/cluster.dashboard-tunnel.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I would say just use `k9s` and drop the kubernetes dashboard. Or use both and use `k9s` to set up the tunnel for you.

Reply:
Thanks for the hint @roschaefer !
I get your point and we will change this in the future. 💫

Please just leave it that way for now and we will modernize all these in the next step.
Cool?

Reply:
If you just need an approval, sure, here you go!